packages/react/src/provider.tsx

```diff
@@ -18,8 +18,14 @@ export const LogtoProvider = ({ logtoConfig, children }: LogtoProviderProperties
     const createClient = async () => {
       try {
         const client = await LogtoClient.create(logtoConfig);
-        dispatch({ type: 'INITIALIZE', isAuthenticated: client.isAuthenticated() });
         setLogtoClient(client);
+        const isLoginRedirect = client.isLoginRedirect(window.location.href);
+        // If is login redirect, should set isLoading and isInitialized at the same time
+        dispatch({
+          type: 'INITIALIZE',
+          isAuthenticated: client.isAuthenticated(),
```

Comment:
usually all dispatch data are wrapped up using a payload object
```js
dispatch({
  type: 'INITIALIZE',
  payload: {
    isAuthenticated,
    isLoading,
  }
})
```

so on the reducer method 

```js
(state, action: { type,  payload }) => { ... }
```